### ISSUE TITLE
Have you seen MakersLink?

### ISSUE THREAD

**Issue by TechnoX** [LINKPing/LINKping#19]
Dear LINKping!

Have you seen the makerspace MakersLink? https://makerslink.se/

We are having a lab in Valla, Linköping. If you are interested I can show you the place?? We have electronics lab, textile corner, a carpentry, 3D printers and more.

**Comment by ralphtheninja** [comment]
I know of the place but haven't visited it yet.

**Comment by TechnoX** [comment]
I would be interested in visiting your place, is it open for a visit maybe during this or next week?

**Comment by ralphtheninja** [comment]
@TechnoX @gnyrfta will be there tomorrow and thursday and I'll be there on thursday this week. Sometime between 11 and 15 or so. Hop onto the irc channel if you feel like it, #linkping on libera chat.

**Comment by gnyrfta** [comment]
Yup, I'm there tomorrow from 11 to 15 - phone # is 073-1447287 .  :  )
